### ISSUE TITLE
Could you create a separate apk to convert psp games to CHD/CSO?

### ISSUE THREAD

**Issue by carlosgamer23** [hrydgard/ppsspp#18311]
### What should happen

Could you create a separate apk to convert psp games to CHD/CSO? To date, there is NO really functional app on Android to extract and compress files into an ISO, much less do we have a "CHDMAN" for Android or an app that compresses and decompresses the CSO.  Could you help with this?  Android is short on resources, the possibility you gave us of using CHD or CSO is good, but we don't have decent programs that are suitable for Android to take advantage of this type of resource, and many have no way of using a PC.  I think it would be wise to have your own app for this (I just don't think it would be so convenient to join PPSSPP, as it would be a lot of work for you, so a separate apk with these functions would be the most sensible)

### Who would this benefit

Many Android users, especially those with simpler cell phones with little available memory.

### Platform (if relevant)

Android

### Games this would be useful in

All games with 2G or more

### Other emulators or software with a similar feature

MAME (I believe they are the creators of CHDMAN, if I'm not mistaken), MedGui (I have a tool for extracting, converting ISO to other formats). But obviously, these functions only exist for Windows, so we need you to perform a miracle.

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by LunaMoo** [comment]
Google for "CISO – PSP ISO Compressor", it's not as good as maxcso, but for android it's more than enough. chd isn't recommended container and it only uses one compression format which is heavy to decompress on lower end devices, not ideal for devices running of battery anyway.

**Comment by carlosgamer23** [comment]
> Google para "CISO – PSP ISO Compressor", não é tão bom quanto maxcso, mas para Android é mais que suficiente. chd não é um contêiner recomendado e usa apenas um formato de compactação que é pesado para descompactar em dispositivos de baixo custo, o que não é ideal para dispositivos que funcionam com bateria.
> 
> Editar: não foi possível encontrar nenhum lugar razoavelmente legítimo para CISO, mas também encontrei https://github.com/RobertAguilera712/cisojr4droid , isso deve ser mais do que suficiente para um usuário Android ser capaz de compactar e descompactar ISOs em formato compactado compatível com PPSSPP . Novamente, faltam recursos do maxCSO, mas você realmente não precisa de mais.

I have already used and created a mod of this app once, it is inefficient and corrupts the games, I do not recommend using it in any way, it is not suitable for a proper conversion, it is safer to use the game in iso than to use CISO apk...

**Comment by NABN00B** [comment]
Sounds like a topic that could get documented on the website.

**Comment by padoverc** [comment]
> for cso, better use this iso tool homebrew bro, https://www.gamebrew.org/wiki/ISO_Tool_PSP
> 
> after you choose file name, it will blackscreen, just close ppsspp to recent app, and back.
> 
> place iso in /psp/iso/

Can this be used to update games? Like take God Eater 2 v1.01 to v1.30?

**Comment by LunaMoo** [comment]
> 
> Can this be used to update games? Like take God Eater 2 v1.01 to v1.30?

No. Not on emulators, due to missing update keys it will just ruin your data. You need real PSP for that to work. One of the reasons why I never recommend this tool, anything native to your platform will be safer and easier to use.

**Comment by padoverc** [comment]
Unrelated, @LunaMoo, did you ever see my issue on your "PPSSPP_workarounds" repo?
